tagstudio/src/core/library.py

```diff
@@ -1091,24 +1028,23 @@ def refresh_dupe_files(self, results_filepath):
         A duplicate file is defined as an identical or near-identical file as determined
         by a DupeGuru results file.
         """
-        full_results_path = (
-            os.path.normpath(f"{self.library_dir}/{results_filepath}")
-            if self.library_dir not in results_filepath
-            else os.path.normpath(f"{results_filepath}")
-        )
-        if os.path.exists(full_results_path):
+        full_results_path = Path(results_path)
```

Comment:
```suggestion
        full_results_path = Path(results_filepath)
```

Reply:
There were a couple other issues with that method it looks like, I fixed it up. Thanks!